scripts/vmbuild.rb

```diff
@@ -129,7 +130,7 @@ def verify_run(output)
     uuid   = verify_run(output)
     $log.info "#{target} base_image complete, uuid: #{uuid}"
 
-    unless vhd_image
+    unless vhd_image || gce_image
```

Comment:
`unless` + `||` is very confusing.  Perhaps instead use `if !vhd_image && !gce_image`?